demisto_sdk/commands/common/errors.py

```diff
@@ -1253,10 +1253,11 @@ def release_notes_config_file_missing_release_notes(config_rn_path: str):
 
     @staticmethod
     @error_code_decorator
-    def release_notes_bc_json_file_missing(json_path):
-        return f"A new release notes file contains the phrase \"breaking changes\" without a proper JSON file.\n" \
-               f"Please add it to the following path: {json_path}\n",\
-               "For more information, refer to the following documentation: https://xsoar.pan.dev/docs/documentation/release-notes"
+    def release_notes_bc_json_file_missing(json_path: str):
+        return f'A new release notes file contains the phrase \"breaking changes\" ' \
+               'without a matching JSON file (with the same name as the release note file, e.g. 1_2_3.md). ' \
+               f'Please run \"demisto-sdk update-release-notes -i {json_path[:-4]}md -bc\". ' \
```

Comment:
Following our conversation, I understand this will not bump the version?

Reply:
Yes.